### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by samtrajten**
Archivos de la segunda entrega.

### CODE REVIEW

File: Assignment2/samtrajten-SamuelDeMorais.rdf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+	<rdf:RDF
```
Comment (ocorcho) [inline Assignment2/samtrajten-SamuelDeMorais.rdf]: You should add a prefix/namespace

File: Assignment2/samtrajten-SamuelDeMorais.ttl [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:Class01 xml:includes :Computer101,
```
Comment (ocorcho) [inline Assignment2/samtrajten-SamuelDeMorais.ttl]: The property xml:includes is not right. Select your own prefix and qName

File: Assignment2/samtrajten-SamuelDeMorais.ttl [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
```
Comment (ocorcho) [inline Assignment2/samtrajten-SamuelDeMorais.ttl]: Add a prefix for your data

File: Assignment2/samtrajten-SamuelDeMorais.ttl [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+:Class01 xml:includes :Computer101,
+    :Sensor029 .
+
+:Computer101 xml:hasOwner :User10A .
+
+:Measurement8401 xml:atTime "2010-06-12T12:00:12"^^xsd:dateTime;
+  xml:hasTemperature "29"^^xsd:int .
+
+:Sensor029 xml:hasMeasurement :Measurement8401 .
```
Comment (ocorcho) [inline Assignment2/samtrajten-SamuelDeMorais.ttl]: same problem as before.

**Comment by samtrajten** [comment]
I've completed the requested changes.

**Pull request merged**